corehq/apps/auditcare/utils/export.py

```diff
@@ -166,7 +166,18 @@ def get_sql_start_date():
     all auditcare data in Couch will be obsolete and/or archived before
     SQL data. It should be removed when the data in Couch is no longer
     relevant.
+
+    NOTE the output is being hardcoded for the time historical auditcare events are copied to SQL
     """
+    if settings.SERVER_ENVIRONMENT == 'production':
+        return datetime(2021, 3, 24, 6, 17, 21, 988840)
```

Comment:
With these dates hard-coded, does it mean that queries of auditcare data may return duplicate records for date ranges that include this date? I assume that is true, and I think it's probably fine, but just confirming that that has been thought about.